### PR TITLE
[Feature] Add session variable to toggle ANSI color in explain output

### LINKED ISSUE #52716: [feature]  SHOW PROFILELIST and analyze feature Further improvement
- [x] https://github.com/StarRocks/starrocks/pull/52960

## Feature request
starrocks version：3.2.10-ee-8946e93
DBeaver：Version 23.1.0.20230604
refer: https://docs.starrocks.io/docs/administration/query_profile_text_based_analysis/

**Describe the solution you'd like**
 commend: `SHOW PROFILELIST [LIMIT <num>]`
can't to filter and sort.if i have too much results，i can‘t find it soon and convenient，and i can't sort it .

**Describe alternatives you've considered**
`SHOW PROFILELIST where Statement like '%logis%' order by startime desc limit 50`
**Additional context**

`ANALYZE PROFILE FROM '21530617-9d78-11ef-b1b2-5254005e63eb'
`analyze commond return too much useless character info
image

### PR DESCRIPTION
**Pull request by merlin83**
## Why I'm doing:
Adds a session variable to toggle ANSI color in explain and analyze output

## What I'm doing:

Fixes #52716

## What type of PR is this:

- [ ] BugFix
- [X] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [x] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.3
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Review by LiShuMing — APPROVED** [review]
+1

**Comment by merlin83** [comment]
Kaisen, given this affects the output for both EXPLAIN and EXPLAIN ANALYZE, I think colorExplainOutput is a better name. What do you think?

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java [hunk 1]
```diff
@@ -966,6 +967,10 @@ public static MaterializedViewRewriteMode parse(String str) {
     @VariableMgr.VarAttr(name = ENABLE_PROFILE, alias = IS_REPORT_SUCCESS)
     private boolean enableProfile = false;
 
+    // Toggle ANSI color in explain output
+    @VariableMgr.VarAttr(name = COLOR_EXPLAIN_OUTPUT)
+    private boolean colorExplainOutput = true;
```
[the remark this answers is not in the document]
Comment (merlin83) [reply fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java:972]: @kangkaisen Given this affects the output for both EXPLAIN and EXPLAIN ANALYZE, I think colorExplainOutput is a better name. What do you think?

**Review by kangkaisen — APPROVED** [review]

**Comment by murphyatwork** [comment]
@Mergifyio rebase

**Comment by murphyatwork** [comment]
@Mergifyio rebase

**Pull request merged**